.prettierrc.yaml

```diff
@@ -0,0 +1,2 @@
+tabWidth: 2
+singleQuote: false
```

Comment:
I added this comment in my own config as a reminder after researching it:
`# in double quotes you can use escapes`